src/Preferences.cc

```diff
@@ -163,10 +163,18 @@ void Preferences::init() {
 	this->defaultmap["advanced/localization"] = true;
 	this->defaultmap["advanced/autoReloadRaise"] = false;
 	this->defaultmap["advanced/enableSoundNotification"] = true;
+	this->defaultmap["advanced/timeThresholdOnRenderCompleteSound"] = 0;
 	this->defaultmap["advanced/enableHardwarnings"] = false;
 	this->defaultmap["advanced/enableParameterCheck"] = true;
 	this->defaultmap["advanced/enableParameterRangeCheck"] = false;
 
+	connect(this->enableSoundOnRenderCompleteCheckBox, SIGNAL(toggled(bool)),
+		        this->timeThresholdOnRenderCompleteSoundLabel, SLOT(setEnabled(bool)));
+	connect(this->enableSoundOnRenderCompleteCheckBox, SIGNAL(toggled(bool)), 
+		        this->secLabel, SLOT(setEnabled(bool)));
+	connect(this->enableSoundOnRenderCompleteCheckBox, SIGNAL(toggled(bool)), 
```

Comment:
Since we use automatic connections for all other controls, we should probably do the same for this.

Reply:
Can you please elaborate. What I understood is that you are expecting a function for SLOT in which state enabled status is toggled.

Reply:
See https://doc.qt.io/qt-5/designer-using-a-ui-file.html#automatic-connections

..and the slots starting here:  https://github.com/openscad/openscad/blob/master/src/Preferences.h#L34